port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -580,6 +582,42 @@ async def sort_and_upsert_failed_entities(self,user_agent_type: UserAgentType)->
             if isinstance(ocean_abort.__cause__,CycleError):
                 for entity in event.entity_topological_sorter.get_entities(False):
                     await self.entities_state_applier.context.port_client.upsert_entity(entity,event.port_app_config.get_port_request_options(),user_agent_type,should_raise=False)
+    async def process_resource(self,resource,index,user_agent_type):
```

Comment:
I feel like we can improve readability and code separation from the `sync_raw` file.